docs/getting-started-guides/minikube.md

```diff
@@ -75,15 +75,37 @@ Stopping local Kubernetes cluster...
 Stopping "minikube"...
 ```
 
-### Using rkt container engine
+### Alternative Container Runtimes
+
+#### CRI-O
+
+To use [CRI-O](https://github.com/kubernetes-incubator/cri-o) as the container runtime, run:
+
+```bash
+$ minikube start \
+    --network-plugin=cni \
+    --container-runtime=cri-o \
+    --bootstrapper=kubeadm
+```
+
+Or you can use the extended version:
+
+```bash
+$ minikube start \
+    --network-plugin=cni \
+    --extra-config=kubelet.container-runtime=remote \
+    --extra-config=kubelet.container-runtime-endpoint=/var/run/crio.sock \
+    --extra-config=image-service-endpoint=/var/run/crio.sock
```

Comment:
you'll need a `--bootstrapper=kubeadm` here also, we just fixed this doc error in the minikube repo.

Reply:
@fntlnz Bumping for review feedback ☝️

Reply:
@fntlnz Deadline for completion is tomorrow, Friday, December 8th.

/cc @idvoretskyi